src/main/java/inspired/pdf/unbox/internal/TextWriter.java

```diff
@@ -76,17 +76,25 @@ public float writeVerticalText(PDPageContentStream contentStream, Bounds bounds,
     }
 
     private float write(List<String> chunks, Bounds bounds, Align align, PDPageContentStream stream, float y) {
-        for (String chunk : chunks) {
+        int index = 0;
+        if(chunks.size() == 0) {
+            return 0;
+        }
+        while(index < chunks.size() && enoughSpace(bounds, index)) {
```

Comment:
As far as I am aware this will only happen when you explicitly set the innerHeight of a Paragraph. Otherwise the Paragraph should claim enough space. But I might not understand all layout dependencies. Maybe there is a case where a paragraph is not granted it's stated innerHeight? But do we really want an overflow to the bottom then?

Reply:
True, an overflow is probably never wanted. But a layout issue would then at least be visible. I think we should merge it that way and maybe add a switch  like overflowVisible() later